docs/reference/NuGet-Client-SDK.md

```diff
@@ -11,11 +11,23 @@ ms.topic: conceptual
 
 The *NuGet Client SDK* refers to a group of NuGet packages:
 
-* [`NuGet.Protocol`](https://www.nuget.org/packages/NuGet.Protocol) - Used to interact with HTTP and file-based NuGet feeds
-* [`NuGet.Packaging`](https://www.nuget.org/packages/NuGet.Packaging) - Used to interact with NuGet packages. `NuGet.Protocol` depends on this package
+* [`NuGet.Indexing`](https://www.nuget.org/packages/NuGet.Indexing) - NuGet's indexing library for the Visual Studio client search functionality.
```

Comment:
Why do we ship this again?
Is it VS4Mac?

Reply:
Great question. [NuGet.org used by tab](https://www.nuget.org/packages/NuGet.Indexing#usedby-body-tab) says https://github.com/mono/monodevelop repo is using that package which is marked as public archive.

Reply:
Added `NuGet.Indexing` package to https://github.com/NuGet/Client.Engineering/issues/2582.

Reply:
monodevelop is what eventually became vs 4 mac, so sounds like yes is the answer.

Reply:
We have at least one person/partner, who I'm confident is not VS4M, using NuGet.Indexing: https://github.com/NuGet/Home/issues/11388

Although given the version number they reported, it sounds like they're getting the package though Arcade/Maestro, not nuget.org. Anyway, if we want to effectively remove this package as part of the client SDK, then we should be aware that it will break some people who are using it.

Reply:
On the contrary, I'm pretty certain those 2 work on VS4M :)